src/Compilers/Core/Portable/InternalUtilities/UnconditionalSuppressMessageAttribute.cs

```diff
@@ -0,0 +1,90 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if !NET5_0_OR_GREATER
+
+using System;
+
+namespace System.Diagnostics.CodeAnalysis;
+
+/// <summary>
+/// Suppresses reporting of a specific rule violation, allowing multiple suppressions on a
+/// single code artifact.
+/// </summary>
+/// <remarks>
+/// <see cref="UnconditionalSuppressMessageAttribute"/> is different than
+/// <see cref="SuppressMessageAttribute"/> in that it doesn't have a
+/// <see cref="ConditionalAttribute"/>. So it is always preserved in the compiled assembly.
+/// </remarks>
+[AttributeUsage(AttributeTargets.All, Inherited = false, AllowMultiple = true)]
+internal sealed class UnconditionalSuppressMessageAttribute : Attribute
```

Comment:
Should use TypeForwardedTo on NET like so: https://github.com/dotnet/roslyn/blob/main/src/Dependencies/Contracts/IsExternalInit.cs

Reply:
But as seen in https://patch-diff.githubusercontent.com/raw/dotnet/roslyn/pull/66519.diff, this file is unused, so maybe we can just delete it.